tools/aws_logs.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
This script was copied entirely from the infrastructure-agent repo (except for the hardcoding of the `region` below) and is used to retrieve the Fargate task logs once they have executed. No need to review it.